.gitlab/package_build/installer.yml

```diff
@@ -135,21 +135,23 @@ updater-arm64:
   needs: ["go_mod_tidy_check", "go_deps"]
   variables:
     PACKAGE_ARCH: arm64
-    DESTINATION_FILE: "datadog-updater_7-arm64.tar.xz"
+    DESTINATION_FILE: "datadog-installer_7-arm64.tar.xz"
     RELEASE_VERSION: "$RELEASE_VERSION_7"
+  before_script:
+    - export RELEASE_VERSION=$RELEASE_VERSION_7
```

Comment:
Do you really need it? Is not  the `variables` declaration enough to have it as an env variable?

Reply:
Nope, this looks like a failure during a rebase or a refactoring leftover.
Thanks for noticing! I removed it